### PR TITLE
Introduce the Media Reference Stack v0.2.0

### PR DESCRIPTION
**Pull request by luisfponce**

### CODE REVIEW

**Comment by luisfponce** [comment]
Hi @DnPlas, hi @Meadosc 

Could you please approve this?
Thanks!

@lsandov1

File: mers/NEWS.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+Changes for `v0.2.0` :
```
Comment (DnPlas) [inline mers/NEWS.md:1]: I guess you would want to change this to v2.0.0

**Review by DnPlas — APPROVED** [review]
LGTM

**Comment by luisfponce** [comment]
DO NOT MERGE until tomorrow please.

File: mers/clearlinux/INSTALL.md [hunk 2]
```diff
@@ -1,49 +1,49 @@
...
-    Since Clear Linux OS is a stateless system, avoid modifying the
-    files under the `/usr` directory instead use `/etc` otherwise the software
-    updater may overwrite `/usr` files.
-
-## Run examples
-
-For transcoding and video analytics examples and pipelines, please see the
-official documentation at: https://clearlinux.org/sw/stacks/media-reference
+# Media Reference Stack
+[![](https://images.microbadger.com/badges/image/clearlinux/stacks-mers.svg)](https://microbadger.com/images/clearlinux/stacks-mers
```
Comment (DnPlas) [inline mers/clearlinux/INSTALL.md]: Maybe change the microbadger as well?

Comment (luisfponce) [reply mers/clearlinux/INSTALL.md]: Nice catch!

@hugosoto @lsandov1 This is like the chicken or the egg dilemma, we do not have currently the other dockerhub place to link it here right?

Comment (luisfponce) [reply mers/clearlinux/INSTALL.md]: a new microbadge has been generated. Thanks.

File: mers/NEWS.md (hunk 1 above)
Comment (unrahul) [inline mers/NEWS.md:1]: Nice release news!

File: mers/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+# Contributing
```
Comment (unrahul) [inline mers/CONTRIBUTING.md:1]: Can we move this to root, and have this as the main contributors guide?

**Review by unrahul — COMMENTED** [review]
Please see the comment.

File: mers/CONTRIBUTING.md (hunk 3 above)
Comment (DnPlas) [reply mers/CONTRIBUTING.md:1]: +1

Comment (luisfponce) [reply mers/CONTRIBUTING.md:1]: @unrahul we have automated Internally the "path files placing" in MeRS, just moving `CONTRIBUTING.md` is no problem ;) but we need to fine tune / adjust our automation to fit this.
Could we merge it as it is, and to follow up this task I can create an issue?

Beside this, I believe it is a good idea to standardize `CONTRIBUTING.md`  -> `root` place and notify other stack folks before doing changes on this kind of files.

Thoughts here @DnPlas @unrahul ?

Comment (DnPlas) [reply mers/CONTRIBUTING.md:1]: Agree, we can do it in a follow on patch.

**Comment by unrahul** [comment]
Merging thank you all.

**Pull request merged**